src/features/surveys/components/ResponseStatsCards.tsx

```diff
@@ -0,0 +1,957 @@
+                  'zetkin-chart-content'
+                );
+              const contentBox =
+                contentOverrides.length > 0
+                  ? contentOverrides[0]
+                  : containerRef.current;
+
+              const boundingRect = contentBox.getBoundingClientRect();
+              doc.body.style.width = `${boundingRect.width}px`;
+              doc.body.style.height = `${boundingRect.height}px`;
+            },
+            type: 'image/png',
+          });
+        } else if (format === 'pdf') {
+          await exportApi.current.exportAsPrint();
+        }
+      } catch (e) {
+        showSnackbar('error', messages.insights.export.errorUnknown());
+      } finally {
+        document.body.style.overflow = docOverflow;
+      }
+    },
+    [
+      showSnackbar,
+      containerRef,
+      exportApi,
+      questionStats.question.question.question,
+    ]
+  );
+
+  const exportMenuItems = useMemo(
+    () => [
+      {
+        label: messages.insights.export.toPng(),
+        onSelect: () => exportChart('png'),
+      },
+      {
+        label: messages.insights.export.toPdf(),
+        onSelect: () => exportChart('pdf'),
+      },
+    ],
+    [exportChart]
+  );
+
+  const [exportMenuAnchorEl, setExportMenuAnchorEl] =
+    useState<null | HTMLElement>(null);
+  const exportMenuOpen = !!exportMenuAnchorEl;
+  const exportMenuHandleClick = useCallback(
+    (event: React.MouseEvent<HTMLButtonElement>) => {
+      setExportMenuAnchorEl(event.currentTarget);
+    },
+    [setExportMenuAnchorEl]
+  );
+  const exportMenuHandleClose = useCallback(() => {
+    setExportMenuAnchorEl(null);
+  }, []);
+
+  return (
+    <ZUICard
+      header={questionStats.question.question.question}
+      status={
+        <Box
+          sx={{
+            alignItems: 'center',
+            display: 'flex',
+            flexDirection: 'row',
+            gap: '5px',
+            height: '100%',
+          }}
+        >
+          <ToggleButtonGroup
+            exclusive
+            onChange={(_, newValue) => newValue && onTabChange(newValue)}
+            orientation={'horizontal'}
+            size={'small'}
+            sx={{
+              '& .MuiToggleButton-root': {
+                '&:hover': {
+                  backgroundColor: theme.palette.grey[100],
+                  color: theme.palette.primary.main,
+                },
+                border: `1px solid ${theme.palette.grey[600]}`,
+                color: theme.palette.primary.main,
+              },
+              '& .MuiToggleButton-root.Mui-selected': {
+                '&:hover': {
+                  color: theme.palette.primary.main,
+                },
+                backgroundColor: theme.palette.primary.main,
+                color: theme.palette.primary.contrastText,
+              },
+              '& .MuiToggleButton-sizeSmall': {
+                fontSize: '12px',
+                padding: '0.2rem 0.4rem',
+              },
+            }}
```

Comment:
What is the purpose of this custom styling? And what would it look like if we just used the default styling?

When I look at this, I think it looks a little odd, with the dark border etc. It looks a little like a mix between what I would expect from MUI (current) and what I would expect from ZUI (future).

<img width="191" height="53" alt="image" src="https://github.com/user-attachments/assets/df987c0c-72ab-44a1-ac48-d3076f9e2555" />